blog/templates/blog/admin/post_detail.html

```
@@ -0,0 +1,51 @@
+  </p>
+
+  <p>
+    작성일: {{ post.created_at }}
+  </p>
+
+  <hr>
+
+  <h2>썸네일</h2>
+
+  {% if post.preview_image %}
+  <img src="{{ post.preview_image.url }}" width="300">
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)